app/views/individual/incometax/business/TaskList.scala.html

```diff
@@ -58,7 +58,11 @@
             <span class="app-task-list__task-completed" id={"section_2_item_1_status"+viewModel.selfEmployments.indexOf(se)}>
              {if(se.confirmed && viewModel.selfEmploymentAccountingMethod.isDefined) {messages("business.task-list.status.completed")} else {messages("business.task-list.status.incomplete")}}
             </span>
+            <span class="govuk-remove-business-link" id="self_employemnt_remove">
```

Comment:
```suggestion
            <span class="govuk-remove-business-link" id={"self_employment_remove"+viewModel.selfEmployments.indexOf(se)}>
```
spelling mistake in "employment" also added an index to identify the individual businesses, as it is, it would be an accessibility issue as if multiple self employments existed, multiple id's would have the same value